src/controller/Demo.java

```diff
@@ -0,0 +1,123 @@
+package controller;
+
+import view.CLI;
+import view.userCommands;
+
+public class Demo {
+
+    public static void main(String[] args) throws Exception {
+
+        int mazeSize = 11;
```

Comment:
Move it lower to where it used

Reply:
Not sure why? all the statements should be before the logic..?

Reply:
Correct for C type of code (as a compiler obligation) but current writing standarts are more for declare where used